src/pages/Archive/Archive.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Sidebar } from '../../components';
+import { useAuth, useData } from '../../context';
+import { deleteArchiveNote, restoreArchiveNote } from '../../services/Services';
+
+export const Archive = () => {
+  const { state, dispatch } = useData();
+  const { token } = useAuth();
+
+  const restoreArchiveHandler = async (note) => {
+    const response = await restoreArchiveNote(note._id, token, note);
+    if (response.status === 200) {
+      dispatch({
+        type: 'ARCHIVE_NOTE',
+        payload: {
+          archiveList: response.data.archives,
+          noteList: response.data.notes,
+        },
+      });
+    }
+  };
+
+  const deleteArchiveHandler = async (note) => {
+    const response = await deleteArchiveNote(note._id, token);
+    if (response.status) {
+      dispatch({
+        type: 'DELETE_ARCHIVE',
+        payload: {
+          archiveList: response.data.archives,
+        },
+      });
+    }
+  };
+
+  return (
+    <>
+      <div className='note__container'>
+        <Sidebar />
+        {state.archiveList.map((note) => {
+          const { _id, title, content, timeCreated, backgroundColor } = note;
+          return (
+            <div
+              style={{ backgroundColor: backgroundColor }}
```

Comment:
can be written as style={{backgroundColor}}

Reply:
Sure I'll update this